nextstep-mvc/src/main/java/nextstep/mvc/DispatcherServlet.java

```diff
@@ -11,36 +14,45 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
 
 @WebServlet(name = "dispatcher", urlPatterns = "/", loadOnStartup = 1)
 public class DispatcherServlet extends HttpServlet {
     private static final long serialVersionUID = 1L;
     private static final Logger logger = LoggerFactory.getLogger(DispatcherServlet.class);
     private static final String DEFAULT_REDIRECT_PREFIX = "redirect:";
 
-    private HandlerMapping rm;
+    private List<HandlerMapping> handlerMappings;
 
-    public DispatcherServlet(HandlerMapping rm) {
-        this.rm = rm;
+    public DispatcherServlet(HandlerMapping manualHandlerMapping, AnnotationHandlerMapping annotationHandlerMapping) {
```

Comment:
Mapping의 개수나 타입에 제한을 두지 않으면 어플리케이션에서 더 편하고 확장성 있게 프레임워크를 사용할 수 있지 않을까요?
```suggestion
    public DispatcherServlet(List<HandlerMapping> handlerMappings) {
```
혹은 
```suggestion
    public DispatcherServlet(HandlerMapping... handlerMappings) {
```